app/facades/merchants_facade.rb

```diff
@@ -0,0 +1,33 @@
+class MerchantsFacade
+
+  attr_reader :service
+
+  def initialize
+    @service = start_service
+  end
+
+  def merchants
+    merchants = service.merchants.map do |data|
+      Merchant.new(data)
+    end
+  end
+
+  def get_merchant(id)
+    merchant = Merchant.new(@service.merchant(id))
+  end
+
+  def get_merchant_items(id)
+    items = @service.merchant_items(id).map do |data|
+      Item.new(data)
+    end
+  end
+
+  def find_merchant(search)
+    merchant = Merchant.new(@service.find_merchant(search)[0])
+  end
+
+  def start_service
+    @service = EngineService.new
+  end
```

Comment:
Similar to the `ItemsFacade`, I might skip the `initialize` and `attr_reader` altogether and do something like:

```ruby
def service
  @service ||= EngineService.new
end
```

Then you could call `service` directly on line 16, 20, and 26 similar to how you are on line 10.